src/evm/protocol/vm/state.rs

```diff
@@ -748,21 +741,21 @@ mod tests {
         let overwrites = pool_state
             .get_overwrites(
                 vec![pool_state.tokens[0], pool_state.tokens[1]],
-                U256::from_big_endian(&(*MAX_BALANCE / rU256::from(100)).to_be_bytes::<32>()),
+                *MAX_BALANCE / U256::from(100),
             )
             .unwrap();
         let dai_limit = pool_state
             .get_sell_amount_limit(vec![dai().address, bal().address], Some(overwrites.clone()))
             .unwrap();
-        assert_eq!(dai_limit, U256::from_dec_str("100279494253364362835").unwrap());
+        assert_eq!(dai_limit, U256::from_str("100279494253364362835").unwrap());
 
         // let bal_limit = pool_state
         //     .get_sell_amount_limit(
         //         vec![pool_state.tokens[1], pool_state.tokens[0]],
         //         Some(overwrites),
         //     )
         //     .unwrap();
-        // assert_eq!(bal_limit, U256::from_dec_str("13997408640689987484").unwrap());
+        // assert_eq!(bal_limit, U256::from_str("13997408640689987484").unwrap());
```

Comment:
Do you know why is this is commented out?

Reply:
good question! I think it was a mistake! it passes anyway! bringing it back! good catch! Thank you 🙏🏼